### PR TITLE
Readme: Add note about TLS flag.

### LINKED ISSUE #36: Unable to connect from other computers
@maffoo, @ejeffrey: When I try to connect to a computer (rottmayer in the example below) running scalabrad-0.5.0 from bintray, I get a message "Expected STARTTLS". Is there something I should be doing (in which case documentation is needed) or is this a bug? Note that I'm running current pylabrad master. Also, this is the first time for running the scalabrad manager on rottmayer, so maybe there's some configuration stuff which needs to be done?

Full traceback as follows:

```
Python 2.7.9 (default, Dec 10 2014, 12:24:55) [MSC v.1500 32 bit (Intel)]
Type "copyright", "credits" or "license" for more information.

IPython 2.1.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import labrad

In [2]: cxn=labrad.connect(host='rottmayer')
---------------------------------------------------------------------------
Error                                     Traceback (most recent call last)
<ipython-input-2-dd6249c2a6d2> in <module>()
----> 1 cxn=labrad.connect(host='rottmayer')

C:\Code\pylabrad\labrad\__init__.pyc in connect(host, port, name, **kw)
     26 def connect(host=constants.MANAGER_HOST, port=constants.MANAGER_PORT, na
me=None, **kw):
     27     """Create a client connection to the labrad manager."""
---> 28     cxn = backend.connect(host=host, port=port, name=name, **kw)
     29     return client.Client(cxn)
     30

C:\Code\pylabrad\labrad\backend.pyc in connect(host, port, name, backend, **kw)
    419             backend = 'asyncore'
    420     cxn = backends[backend](name)
--> 421     cxn.connect(host, port, **kw)
    422     return cxn
    423

C:\Code\pylabrad\labrad\backend.pyc in connect(self, host, port, timeout, passwo
rd)
     33         self.host = host
     34         self.port = port
---> 35         self.ID = self._connect(password, timeout)
     36         self.connected = True
     37

C:\Code\pylabrad\labrad\backend.pyc in _connect(self, password, _timeout)
     62         def _connect(self, password, _timeout):
     63             startReactor()
---> 64             self.cxn = self.call(getConnection, self.host, self.port, se
lf.name, password).wait()
     65             return self.cxn.ID
     66

C:\Code\pylabrad\labrad\backend.pyc in wait(self)
    396                 # If any Future in the queue fails,
    397                 # we immediately bail.
--> 398                 result.raiseException()
    399             else:
    400                 for func, args, kw in f.callbacks:

C:\Python279\lib\site-packages\twisted\python\failure.pyc in raiseException(self
)

Error: (2) Expected STARTTLS [payload=None]

In [3]:
```

### PR DESCRIPTION
**Pull request by jwenner**
Add note to readme to explain how to have non-TLS-enabled sessions connect.

Fixes #36, #37

### CODE REVIEW

**Comment by maffoo** [comment]
Thanks for this addition. Maybe also just add a note to encourage people to invoke the manager with the `--help` option to see documentation of all the available command line and environment variable configuration options. We worked hard to add those docs, and that will in general be the authoritative place to get information for a given manager binary.

File: README.md [hunk 1]
```diff
@@ -46,3 +46,5 @@ Simply download and install the archive file for the version you want and unpack
 For example, if you download [scalabrad-0.3.1.tar.gz](https://bintray.com/artifact/download/labrad/generic/scalabrad-0.3.1.tar.gz) and untar it in some directory `$PATH/`, then you can run `$PATH/scalabrad-0.3.1/bin/labrad` to start the manager. You'll need to have Java 8 installed on your system, but all other dependencies are included in the package. Windows `.bat` files are also included in the distribution, though you may need a tool like 7-Zip to extract the `.tar.gz` archive.
```
Comment (maffoo) [inline README.md]: While you're in here making some changes, maybe update these links to point to 0.5.0 instead of 0.3.1.

**Comment by maffoo** [comment]
A couple minor comments, otherwise LGTM.

**Comment by jwenner** [comment]
@maffoo, how does this look now? Thanks.

**Comment by maffoo** [comment]
LGTM

**Pull request merged**